contrib/ci/Dockerfile-debian.in

```diff
@@ -23,4 +23,4 @@ RUN echo fubar > /etc/machine-id
 %%%ARCH_SPECIFIC_COMMAND%%%
 %%%INSTALL_DEPENDENCIES_COMMAND%%%
 WORKDIR /github/workspace
-CMD ["./contrib/ci/debian.sh"]
+CMD ["./contrib/ci/debian.sh"%%%QUBES%%%]
```

Comment:
rather than an argument, can this be done via an environment variable?  That's how we pass `CI` and `CI_NETWORK` and such in.

Reply:
Deleted. I've added the environment variable `QUBES=true`.